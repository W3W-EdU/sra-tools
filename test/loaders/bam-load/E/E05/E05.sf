#bam-load error test #05 : an aligment lines up past the end of the reference

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E05.fasta

# write the final SAM-output into this file
sam-out:E05.sam

# write a config-file for bam-load
config:E05.config

#4990 + 30 --> past the end of the reference...
p:name=A1,pos=4990
p:name=A1,pos=300

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50

