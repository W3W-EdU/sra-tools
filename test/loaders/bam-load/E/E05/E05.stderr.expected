warn: libs/align/writer-reference.c:3019:ReferenceSeq_Compress: data invalid while processing file within alignment module - Spot 'A1': bad alignment to reference 'R1' at 4989
